src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs

```diff
@@ -224,6 +224,11 @@ public override void VisitLocalFunctionStatement(LocalFunctionStatementSyntax no
                 var oldMethod = _containingMemberOrLambda;
                 _containingMemberOrLambda = match;
 
+                foreach (var parameter in node.ParameterList.Parameters)
+                {
+                    Visit(parameter.Default);
```

Comment:
>Visit(parameter.Default); [](start = 20, length = 25)

Looking at SourceComplexParameterSymbol.MakeDefaultExpression:
```
var convertedExpression = binder.CreateBinderForParameterDefaultValue(this, defaultSyntax).BindParameterDefaultValue(defaultSyntax, parameterType, diagnostics, out valueBeforeConversion);
```

It feels like we need to make sure that whatever we do here is consistent (for example, BinderFlags.ParameterDefaultValue flag is set appropriately). Also, SourceComplexParameterSymbol.MakeDefaultExpression is going to create its own binder instead of using the one created by LocalBinderFactory. This looks suspicious.